packit/api.py

```diff
@@ -2139,6 +2143,107 @@ def run_osh_build(
 
         return cmd_result.stdout
 
+    def create_obs_project(
+        self,
+        project: str,
+        targets: str,
+        owner: Optional[str],
+        package_config: PackageConfig,
+        description: Optional[str],
+    ):
+        conf.get_config()
+        owner = (
+            owner or conf.config["api_host_options"][self.obs_helper._API_URL]["user"]
+        )
```

Comment:
What about putting this into the `obs_helper` so we have as little code here as possible? What do you think?

Reply:
If possible, we could move all the technical/implementation details from this method as well.